spec/models/api_azure_search_spec.rb

```diff
@@ -365,35 +363,47 @@
       end
     end
 
-    skip '#as_json' do
+    describe '#as_json' do
       subject(:search) do
-        agency = Agency.create!({name: 'Some New Agency', abbreviation: 'SNA' })
+        agency = Agency.create!({ name: 'Some New Agency', abbreviation: 'SNA' })
         AgencyOrganizationCode.create!(organization_code: 'XX00', agency: agency)
         affiliate.stub(:agency).and_return(agency)
 
-        described_class.new search_params
+        described_class.new(search_params)
       end
 
-      before { search.run }
+      before do
+        # SRCH-3125: Yes, this is being stubbed with a BingV7 response, not BingV5, but it is what we already have in
+        # the code base. Since Azure/BingV5 are slated for removal, this was the lightest lift to merely ensure
+        # adequate code coverage during the removal of Google.
+        results = Rails.root.join('spec/fixtures/json/bing_v7/web_search/gov.json').read
+        stub_request(:get, /#{BingV5WebEngine::API_HOST}.*gov/o).
+          to_return(status: 200, body: results)
+        search.run
+      end
 
       it 'returns results' do
-        expect(search.as_json[:web][:results].count).to eq(20)
+        result = Hashie::Mash.new(search.as_json[:web][:results].first)
```

Comment:
Added for code coverage.  This and ln 366 are the only real changes here, the rest is CodeClimate inspired style changes.